Lib/test/test_warnings/__init__.py

```diff
@@ -877,6 +877,18 @@ def test_showwarning(self):
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
 
+        # bpo-35178: Test custom formatwarning can receive line as positional
+        def formatwarning(message, category, filename, lineno, text):
```

Comment:
nitpick: use a different function name to avoid confusion, ex: "def myformatwarning"